src/core/expr/fexpr_fillna.cc

```diff
@@ -104,13 +104,12 @@ class FExpr_FillNA : public FExpr_Func {
               indices[i] = static_cast<int32_t>(fill_id);
             }
           } else {
-            for (size_t i = i1; i < i2; ++i) {
-              is_valid = col.get_element(i, &value);
-              fill_id = is_valid? i : fill_id;
-              indices[i] = static_cast<int32_t>(fill_id);
+              for (size_t i = i1; i < i2; ++i) {
+                is_valid = col.get_element(i, &value);
+                fill_id = is_valid? i : fill_id;
+                indices[i] = static_cast<int32_t>(fill_id);
+              }
```

Comment:
and this one?

Reply:
same here as well, aligning the `}` bracket under the `for`